src/ray/object_manager/object_manager.h

```diff
@@ -311,6 +311,9 @@ class ObjectManager : public ObjectManagerInterface,
 
   void Tick(const boost::system::error_code &e);
 
+  /// Get the current object store memory usage.
+  int64_t GetUsedMemory() const { return used_memory_; }
```

Comment:
Can we actually report based on the available memory that the PullManager uses instead? Maybe we should be reporting both, but I think it's a bit misleading to report this one since it includes unpinned objects.

Reply:
Discussed offline: I think "actually used" memory is slightly preferable, but we might want to revisit in the future.